config/artsey.dtsi

```diff
@@ -213,6 +219,10 @@
 #define LAYER_ID_BIG_SYM 7
 #define LAYER_ID_BIG_FUNCTION 8
 
+#if !defined COMBO_LAYER_FILTER
+#define COMBO_LAYER_FILTER 0 1 2 3 4 5 6 7 8
+#endif
+
```

Comment:
Can you rename this to 'COMBO_LAYER_FILTER_ALL' so its clear this is all layers, Also it should use the `LAYER_ID*` values instead of being hard coded.

Reply:
Are the other combo filters defined? If not, can you also refactor them so they can be overridden on a per combo basis?

Reply:
This is a filter just for the global combos. Maybe I should call it `GLOBAL_COMBO_LAYER_FILTER`?

Reply:
Upon further thought, this code can be remved. See comment(s) below in the combo definitons